### PR TITLE
Add multiple assignment mutations

### PR DESCRIPTION
**Pull request by backus**
This PR adds several new mutations for parallel assignment

### CODE REVIEW

**Comment by backus** [comment]
@dkubb @mbj ready for review

File: meta/lvasgn.rb [hunk 1]
```diff
@@ -6,3 +6,28 @@
...
+
+Mutant::Meta::Example.add :lvasgn do
+  source 'a = nil; a'
+
+  mutation 'a = nil; nil'
+  mutation 'a = nil; self'
+  mutation 'a = nil'
+  # TODO: fix invalid AST
+  #   These ASTs are not valid and should NOT be emitted
+  #   Mutations of lvarasgn need to be special cased to avoid this.
```
Comment (mbj) [inline meta/lvasgn.rb]: I think we should not add more "invalid AST emitted" to the code base. Lets fix the lvasgns first?

Comment (backus) [inline meta/lvasgn.rb]: Is the issue here that we shouldn't replace an assignment with a noop in general? Or is it a question of not eliminating the assignment when we can statically check if the lvar is used below?

Comment (mbj) [inline meta/lvasgn.rb]: > Is the issue here that we shouldn't replace an assignment with a noop in general? Or is it a question of not eliminating the assignment when we can statically check if the lvar is used below?

I was woundering if we should not add to the "mutation operator can cause invalid AST but correct source" AKA "unintend source" problem, and instead fix it before.

**Comment by mbj** [comment]
Closing this PR, as all contributions will require a CLA from now on.

**Pull request closed without merging**